docs/guides-buidler-migration.md

```diff
@@ -0,0 +1,193 @@
+
+module.exports = {
+  defaultNetwork: 'localhost',
+  networks: {
+    localhost: {
+      url: 'http://localhost:8545'
+    },
+  },
+  solc: {
+    version: '0.4.24',
+    optimizer: {
+      enabled: true,
+      runs: 10000
+    }
+  },
+  aragon: {
+    appServePort: 1234,
+    clientServePort: 3000,
+    appSrcPath: 'app/',
+    appBuildOutputPath: 'dist/',
+    hooks: require('./scripts/buidler-hooks')
+  }
+}
+```
+
+You can find more information about the Buidler configuration file [here](https://buidler.dev/config). As for the `aragon` section, it contains 5 main options:
+
+  - `appServePort`: The app server port started by the `serve` script of the `app` folder. (Default: `1234`)
+  - `clientServePort`: Aragon client server port.
+  - `appSrcPath`: Frontend source for the app.
+  - `appBuildOutputPath`: Built app path.
+  - `hooks`: Aragon hooks functions. (See [Hooks](#5-hooks) section for more information)
+
+## 5. Hooks
+
+Hooks are custom functions called at various stages of an app initialization. They can be used to execute any logic before or after the app is created and also to specify the app's [`initialize()`](https://hack.aragon.org/docs/aragonos-building#constructor-and-initialization) function parameters. For example, initializing the app with a `uint` and a `string` would be as simple as writing the following hook:
+
+```js
+  getInitParams: async ({}, { web3, artifacts }) => {
+    return [23, "Hello, World!"]
+  }
+```
+
+Hooks can also be used to replace Aragon's templates logic. You can find complete examples in 1Hive's [Conviction Voting](https://github.com/aragonone/conviction-voting-app/blob/buidlerized/buidler-app/scripts/buidler-hooks.js) and [Payroll](https://github.com/1Hive/payroll-app/blob/buidler-setup/buidler-app/scripts/buidler-hooks.js) apps. In our case, we will simply add empty functions since we don't need any initialization logic. So we will create a file named `buidler-hooks.js` in the `scripts` folder and add the following code:
+
+```js
+module.exports = {
+
+  // Called before a dao is deployed.
+  preDao: async ({}, { web3, artifacts }) => {},
+
+  // Called after a dao is deployed.
+  postDao: async ({ dao }, { web3, artifacts }) => {},
+
+  // Called after the app's proxy is created, but before it's initialized.
+  preInit: async ({ proxy }, { web3, artifacts }) => {},
+
+  // Called after the app's proxy is initialized.
+  postInit: async ({ proxy }, { web3, artifacts }) => {},
+
+  // Called when the start task needs to know the app proxy's init parameters.
+  // Must return an array with the proxy's init parameters.
+  getInitParams: async ({}, { web3, artifacts }) => {
+    return []
+  },
+
+  // Called after the app's proxy is updated with a new implementation.
+  postUpdate: async ({ proxy }, { web3, artifacts }) => {}
+}
+```
+
+## 5. Migrate tests 
+
+Since the Aragon Buidler plugin uses truffle 5, you may have to migrate part of your tests. Here are a few tips to ease the process.
```

Comment:
I believe there was a blog post by truffle about this migration when they first released, but otherwise, there should be lots of good guides that we can link to :).